### PR TITLE
Couple of e2e FFDC improvements

### PR DESCRIPTION
**Pull request by 2uasimojo**
- Make hive-operator use `debug` log level
- Save the HiveConfig

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -202,7 +202,7 @@ deploy: ensure-kustomize install
 	mkdir -p overlays/deploy
 	cp overlays/template/kustomization.yaml overlays/deploy
 	cd overlays/deploy && ../../$(KUSTOMIZE) edit set image registry.ci.openshift.org/openshift/hive-v4.0:hive=${IMG} && ../../$(KUSTOMIZE) edit set namespace ${HIVE_OPERATOR_NS}
-	$(KUSTOMIZE) build overlays/deploy | oc apply -f -
+	$(KUSTOMIZE) build overlays/deploy | sed 's/        - info/        - debug/' | oc apply -f -
```
Comment (dlom) [inline Makefile:205]: I wish there was a more robust way to do this, but it's probably good enough as is

**Review by dlom — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (2uasimojo) [reply Makefile:205]: Yeah. I looked into using kustomize, but its `edit` subcommand is super limited. I did verify that this was the only hit for `info` (never mind the spaces and hyphen) in the whole manifest.

**Comment by 2uasimojo** [comment]
@dlom forgot about post-cleanup. Fixed.

**Pull request merged**